script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -523,32 +569,36 @@ No. That can't be right. It's true that I was afraid. That's an undeniable fact.
 [sha:49f5cf36f4ec399dbee9658487a3627bc563f12f]{
 -- Page 664, Offset 26039.
 -- あいつはいつだって俺の選択を尊重していた。
-She always respected my choices.
+She's always respected my choices.
 }
 [sha:aefe449981c9a0796a80e8725b9c922c2a88236f]{
 -- Page 664, Offset 26040.
 -- 力任せなんて、一度もなかったんだ。
-She has never forced me to comply, not even once.
+She's never forced me to comply, not even once.
 }
 [sha:32d4f26942777345f952d20ddf0c27d7065ccb34]{
 -- Page 664, Offset 26041.
 -- 俺がアルクェイドと一緒にいたのは、他の誰でもない、俺自身がそうしたかっただけの話だ……！
-The reason I was with Arcueid was not because of anyone else's will, but because I just wanted to be with her...!
+I didn't stay together with Arcueid because of anyone's will other than my own.
```

Comment:
Wouldn't "anyone else's will other than my own" suggest his will is a member of the set of other peoples' wills?

Reply:
I don't follow to be quite honest.
> I didn't stay together with Arcueid because of anyone else's will.

Reply:
Oh, I misinterpreted as `s/anyone's/anyone else's` but keeping the rest
> I didn't stay together with Arcueid because of anyone else's will other than my own. 

rather than reverting that part of the sentence entirely